src/drivers/serac.cpp

```diff
@@ -154,10 +112,9 @@ int main(int argc, char* argv[])
   std::set<int> trac_bdr = {2};
 
   // loading parameters
-  double tx, ty, tz;
-  inlet->get("nonlinear_solid/tx", tx);  // has default value
-  inlet->get("nonlinear_solid/ty", ty);  // has default value
-  inlet->get("nonlinear_solid/tz", tz);  // has default value
+  double tx = (*inlet)["nonlinear_solid/tx"];  // has default value
```

Comment:
These should be a struct.  Though I don't know what to name it, do we need a generic `Point` struct? @jamiebramwell ?

Reply:
This is really a generic vector of length dimension of the problem. I think a generic vector struct of three (or two) doubles makes sense.

Reply:
I went ahead and added a conversion from inlet to `mfem::Vector`, though I'm not 100% sure where it belongs - I put it in `infrastructure/input.*pp` for now because it's somewhat input related.